app/services/RestPdfGenerator.java

```diff
@@ -35,7 +39,7 @@ public RestPdfGenerator(Config configuration, WSClient wsClient) {
                 "values", values
         );
 
-        Logger.info("Generating PDF: " + request);
+        Logger.info("Generating PDF: {} {}", values, asLoggableLine(values));
```

Comment:
So log the cut down version of toString() and all the fields with just field lengths for the Strings